scripts/tools/EXT_Exp_MetricProgrammable.rst

```diff
@@ -0,0 +1,120 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    t=tags['$t']
+    T=t.upper()
+%>
+:orphan:
+
+.. _ZET_experimental_programmable_metric:
+
+==========================================
+Metric Programmable Experimental Extension
+==========================================
+
+API
+----
+* Enumerations
+
+    * ${t}_metric_programmable_param_type_exp_t
```

Comment:
you would also need to enumerate `${t}_metric_programmable_exp_version_t`

Reply:
Ok .. Will add that.
Thanks